### PR TITLE
docs: fix "write your first package" curl commands

### PR DESCRIPTION
**Pull request by leovct**
## Description:

- Fix some of the curl commands in the docs by adding a space between `curl` and the verb (e.g. `curl -XGET` becomes `curl -X GET`).
- A bunch of automatic linter updates (that I can remove if needed).

## Is this change user facing?
YES

### CODE REVIEW

File: docs/docs/get-started/write-your-first-package.md [hunk 1]
```diff
@@ -703,13 +703,13 @@ You can paste the URL from your output into your browser (or Cmd+click if you're
...
 ```bash
-curl -XPOST -H "content-type: application/json" http://127.0.0.1:$YOUR_PORT/actor --data '{"first_name": "Kevin", "last_name": "Bacon"}'
+curl -X POST -H "content-type: application/json" http://127.0.0.1:$YOUR_PORT/actor --data '{"first_name": "Kevin", "last_name": "Bacon"}'
 ```
 
 ...and then query for it (again replacing `$YOUR_PORT` with your port)...
 
 ```bash
-curl -XGET "http://127.0.0.1:$YOUR_PORT/actor?first_name=eq.Kevin&last_name=eq.Bacon"
+curl -X GET "http://127.0.0.1:$YOUR_PORT/actor?first_name=eq.Kevin&last_name=eq.Bacon"
```
Comment (leovct) [inline docs/docs/get-started/write-your-first-package.md:710]: Main changes :)

File: docs/docs/get-started/write-your-first-package.md [hunk 2]
```diff
@@ -521,9 +521,9 @@ An error occurred executing instruction (number 4) at DEFAULT_PACKAGE_ID_FOR_SCR
   09/May/2023:19:18:41 +0000: Attempting to connect to the database...
   09/May/2023:19:18:41 +0000: {"code":"PGRST000","details":"connection to server at \"10.1.0.3\", port 5432 failed: FATAL:  password authentication failed for user \"postgres\"\n","hint":null,"message":"Database connection error. Retrying the connection."}
   09/May/2023:19:18:41 +0000: connection to server at "10.1.0.3", port 5432 failed: FATAL:  password authentication failed for user "postgres"
-  
+
```
Comment (h4ck3rk3y) [inline docs/docs/get-started/write-your-first-package.md]:
```suggestion
```

File: docs/docs/get-started/write-your-first-package.md [hunk 3]
```diff
@@ -521,9 +521,9 @@ An error occurred executing instruction (number 4) at DEFAULT_PACKAGE_ID_FOR_SCR
...
   postgrest: thread killed
-  
+
```
Comment (h4ck3rk3y) [inline docs/docs/get-started/write-your-first-package.md]:
```suggestion
```

this is output from Kurtosis; pasted as is

**Review by h4ck3rk3y — APPROVED** [review]

File: docs/docs/get-started/write-your-first-package.md (hunk 1 above)
Comment (h4ck3rk3y) [reply docs/docs/get-started/write-your-first-package.md:710]: Thanks @leovct for the cleanup :-)

**Pull request merged**